app/(pages)/_components/SelectButton.tsx

```diff
@@ -0,0 +1,30 @@
+import useSelectMode from '../_hooks/useSelectMode';
+
+const SelectButton: React.FC = () => {
+  const { selectMode, toggleSelectMode } = useSelectMode();
+
+  const buttonStyle = {
+    fontSize: '12px',
+    padding: '6px 10px',
+    border: '1px solid #ccc',
+    borderRadius: '4px',
+    backgroundColor: selectMode ? '#f0f0f0' : 'transparent',
+    color: selectMode ? '#333' : '#000',
+    cursor: 'pointer',
+  };
+
+  const handleClick = () => {
```

Comment:
if all handleClick does is call another function, then you can just write the function in the onClick